glusterd2/commands/volumes/commands.go

```diff
@@ -32,6 +32,18 @@ func (c *Command) Routes() route.Routes {
 			Pattern:     "/volumes/{volname}/options",
 			Version:     1,
 			HandlerFunc: volumeOptionsHandler},
+		route.Route{
+			Name:        "VolumeProfileTunables",
```

Comment:
Users can confuse this name with the [volume profile](http://docs.gluster.org/en/latest/Administrator%20Guide/Monitoring%20Workload/) command which records the fops and collects I/O metrics.

How about `/volumes/group-options` endpoint ? Avoid using profile elsewhere too.

@kshlm @aravindavk Thoughts ?